uv.gyp

```diff
@@ -108,6 +108,7 @@
             'libraries': [
               '-ladvapi32',
               '-liphlpapi',
+              '-lole32',
```

Comment:
You also need to add it here: https://github.com/libuv/libuv/blob/v1.x/Makefile.am#L48